torch/csrc/jit/codegen/cuda/scheduler/pointwise.cpp

```diff
@@ -249,20 +249,6 @@ c10::optional<PointwiseParams> getPointwiseHeuristics(
           TensorDomain::noBroadcasts(largest_out->domain()->domain()))
               .size() == 0 ||
       n_elems == 0) {
-    auto vectorizable_inputs_outputs_entry = HeuristicSummaryEntry<
-        HeuristicCompileTime::VectorizableInputsAndOutputs>(data_cache, []() {
-      return std::make_unique<std::vector<TensorView*>>();
-    });
-    vectorizable_inputs_outputs_entry.get();
-
-    auto broadcast_byte_multiples_entry =
-        HeuristicSummaryEntry<HeuristicCompileTime::BroadcastMultiples>(
-            data_cache, []() {
-              return std::make_unique<
-                  std::vector<scheduler_utils::BroadcastMultiple>>();
-            });
-    broadcast_byte_multiples_entry.get();
-
```

Comment:
@shmsong I don't know why this code was placed here. Is there any nuances to using the cache where we would want to make sure we have an entry even if the schedule is effectively a null schedule?

Reply:
I think we should just split out a Null Scheduler.
The issue is pointwise scheduler is really overloaded, and we should have a NullScheduler for scalar fusion (fusion of size==1 tensors), and a NullShcheduler for null fusions (size == 0, or output == input).
Until we do that though I don't think we can remove these lines, can we?